README.md

```diff
@@ -21,7 +21,10 @@ Uses Redux store, actions and reducers in React.
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://62e6137ae62a5c000904feed--statuesque-truffle-71704f.netlify.app/home)
+
+[Live Demo Link](https://magnificent-granita-6be4b7.netlify.app/home)
```

Comment:
- [ ] Kindly be noticed that the link to the live demo is broken, please fix that.

![Screenshot from 2022-08-02 01-25-59](https://user-images.githubusercontent.com/33752735/182261651-b46a9527-9adb-468c-8945-31e15e753174.png)